bubble_sort/Python/bubble_sort.py

```diff
@@ -0,0 +1,21 @@
+def bubble_sort(arr):
+    """
+    Function to sort a list of integers using bubble sort algorithm
+    """
+    for i in range(len(arr)-1):
+        for j in range(len(arr)-i-1):
+            if arr[j]>arr[j+1]:
+                tmp = arr[j]
+                arr[j] = arr[j+1]
+                arr[j+1] = tmp
+    return arr
+
+def main():
+    arr = [3, 1, 4, 2, 5]
+    sorted_arr = bubble_sort(arr)
+    print("Sorted array: ")
+    for i in sorted_arr:
+        print(i, end=" ")
+
+if __name__ == '__main__':
+    main()
```

Comment:
Line contains following spacing inconsistencies:
- No newline at EOF.

*Origin: SpaceConsistencyBear, Section: `all.pyjava`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp2nd9__kg/bubble_sort/Python/bubble_sort.py
+++ b/tmp/tmp2nd9__kg/bubble_sort/Python/bubble_sort.py
@@ -18,4 +18,4 @@
         print(i, end=" ")
 
 if __name__ == '__main__':
-    main()+    main()
```